extensions/whisper-it/README.md

```diff
@@ -0,0 +1,3 @@
+# Whisper It
+
+Open source whisper extension
```

Comment:
Needs a better `README` that explains how it works and how to set it up, as not many has what it requires to use this and I don't even know what I need 😊